corehq/apps/sso/configuration.py

```diff
@@ -71,7 +71,7 @@ def _get_advanced_saml2_settings(identity_provider):
             "wantNameId": True,
             "wantMessagesSigned": False,  # Entra ID does not support this, premium or standard
             "wantNameIdEncrypted": False,  # Entra ID will not accept if True
-            "failOnAuthnContextMismatch": True,  # very important
+            "failOnAuthnContextMismatch": False,  # IdP ensures auth but cannot guarantee a particular context
```

Comment:
Would be nice to know the original author's reason for `# very important`. I didn't catch it if you explained it in the PR description.